docs/api.rst

```diff
@@ -2,35 +2,44 @@ API documentation
 ~~~~~~~~~~~~~~~~~
 
 .. autoclass:: pytket.extensions.qiskit.IBMQBackend
-    :special-members:
+    :special-members: __init__
+    :show-inheritance:
     :members:
 
 .. autoclass:: pytket.extensions.qiskit.IBMQEmulatorBackend
     :special-members: __init__
+    :show-inheritance:
     :members:
 
 .. autoclass:: pytket.extensions.qiskit.AerBackend
     :special-members: __init__
+    :show-inheritance:
     :inherited-members:
     :members:
 
 .. autoclass:: pytket.extensions.qiskit.AerStateBackend
     :special-members: __init__
     :inherited-members:
+    :show-inheritance:
     :members:
 
 .. autoclass:: pytket.extensions.qiskit.AerUnitaryBackend
     :special-members: __init__
+    :show-inheritance:
     :inherited-members:
     :members:
 
 .. automodule:: pytket.extensions.qiskit
     :members: qiskit_to_tk, tk_to_qiskit, process_characterisation
 
 .. automodule:: pytket.extensions.qiskit.tket_backend
+    :show-inheritance:
     :members:
     :special-members: __init__
 
+.. automodule:: pytket.extensions.qiskit.backends.crosstalk_model
```

Comment:
I've added the three classes `CrosstalkParams`, `NoisyCircuitBuilder` and `FractionalUnitary`. 

These are the only three classes which have tests so I assumed they were the only user facing classes. Is that correct?

Reply:
Update: Only `CrosstalkParams` is user facing